Dockerfile.worker

```
@@ -9,6 +9,8 @@ ENV LANG=en_US.UTF-8 \
     HOME=/home/packit \
     PS_PATH=/src-packit-service
 
+RUN dnf install -y ansible && dnf clean all -y
```

Don't clean the `dnf` cache here, otherwise, the next step (`ansible-playbook files/install-deps-worker.yaml`) will need to download all the metadata again.

```suggestion
RUN dnf install -y ansible
```

---

Or merge those two `RUN` commands to have it in one layer and not saved in the final image. (And move the `COPY` before that.)

---

fixed